os/net/netmgr/net_vfs.c

```diff
@@ -178,8 +178,8 @@ int net_ioctl(int sd, int cmd, unsigned long arg)
 	}
 
 	/* ToDo:  Verify that the sd corresponds to valid, allocated socket */
-	sock = get_socket(sd);
-	if (NULL == sock) {
+	sock = get_socket(sd, getpid());
+	if (sock == NULL) {
```

Comment:
why did you change order?

Reply:
because i think it looks better than earlier one.